README.md

```diff
@@ -112,8 +106,14 @@ holmes investigate alertmanager --alertmanager-url http://localhost:9093
 #  holmes investigate alertmanager --alertmanager-url http://docker.for.mac.localhost:9093
 ```
 
-To investigate alerts in your browser, sign up for a free trial of [Robusta SaaS](https://platform.robusta.dev/signup/?utm_source=github&utm_medium=holmesgpt-readme&utm_content=ways_to_use_holmesgpt_section).
+<b>To investigate alerts in your browser, sign up for a free trial of [Robusta SaaS](https://platform.robusta.dev/signup/?utm_source=github&utm_medium=holmesgpt-readme&utm_content=ways_to_use_holmesgpt_section). </b>
 
+
+<b>Optional:</b> port-forward to AlertManager (if running Prometheus inside Kubernetes)
```

Comment:
can you makeit more clear you should run this *before* running the above command?